src/Plugin/dgi_migrate_alter/spreadsheet/DssiNodeAlter.php

```diff
@@ -0,0 +1,225 @@
+<?php
+
+namespace Drupal\bcelndora\Plugin\dgi_migrate_alter\spreadsheet;
+
+use Drupal\Component\Utility\NestedArray;
+use Drupal\dgi_migrate_alter\Plugin\MigrationAlterBase;
+use Drupal\dgi_migrate_alter\Plugin\MigrationAlterInterface;
+
+/**
+ * Alter for dssi_node migration.
+ *
+ * @MigrationAlter(
+ *   id = "bcelndora_dssi_node_alter",
+ *   label = @Translation("DSSI Node Migration Alteration"),
+ *   description = @Translation("Alters the DSSI Node migration."),
+ *   migration_id = "dssi_node"
+ * )
+ */
+class DssiNodeAlter extends MigrationAlterBase implements MigrationAlterInterface {
+
+  /**
+   * {@inheritdoc}
+   *
+   * @throws \JsonException
+   */
+  public function alter(array &$migration): void {
+    $process =& $migration['process'];
+
+    $process['field_resource_type'][1]['bundle'] = 'library_of_congress_resource_typ';
+
+    // Add Text (plain) delimited fields.
+    $plain_text_delimited_fields_to_add = [
+      'field_local_contexts' => ['local_contexts', ';'],
+      'field_access_id' => ['access_id', ';'],
+    ];
+
+    foreach ($plain_text_delimited_fields_to_add as $field => $field_info) {
+      [$source, $delimiter] = $field_info;
+      $process[$field] = [
+        [
+          'plugin' => 'skip_on_empty',
+          'source' => $source,
+          'method' => 'process',
+        ],
+        [
+          'plugin' => 'dgi_migrate.process.explode',
+          'delimiter' => $delimiter,
+        ],
+        [
+          'plugin' => 'skip_on_empty',
+          'method' => 'row',
+          'message' => "No usable values in $field[1].",
+        ],
+      ];
+    }
+
+    // Add Text (plain) non-delimited fields.
+    $plain_text_non_delimited_fields_to_add = [
+      'field_scale' => 'scale',
+    ];
+
+    foreach ($plain_text_non_delimited_fields_to_add as $field => $source) {
+      $process[$field] = [
+        [
+          'plugin' => 'skip_on_empty',
+          'source' => $source,
+          'method' => 'process',
+        ],
+      ];
+    }
+
+    // Add tertiary delimiters.
+    $tetiary_delimiter_fields_to_add = [
+      ['field_origin_information', 'field_place', 'place', '^'],
+      ['field_origin_information', 'field_publisher', 'publisher', '^'],
+      ['field_origin_information', 'field_edition', 'edition', '^'],
+      ['field_origin_information', 'field_issuance', 'issuance', '^'],
+      ['field_origin_information', 'field_frequency', 'frequency', '^'],
+    ];
+    $expected_child_fields = [
+      'field_event_type', 'field_place', 'field_date_created',
+      'field_date_issued', 'field_date_captured', 'field_date_valid', 'field_date_modified',
+      'field_other_date', 'field_copyright_date', 'field_publisher', 'field_edition',
+      'field_issuance', 'field_frequency',
+    ];
+
+    foreach ($tetiary_delimiter_fields_to_add as $field) {
+      $parent_field = $field[0];
+      $child_field = $field[1];
+      $source_key = $field[2];
+      $delimiter = $field[3];
```

Comment:
Not sure, is array unpacking easier to read?:

```suggestion
      [
        $parent_field,
        $child_field,
        $source_key,
        $delimiter,
      ] = $field;
```